public/js/app/components/ListYears.jsx

```diff
@@ -3,56 +3,137 @@ import Details from './Details'
 import DocumentLinksNav from './DocumentLinksNav'
 import TableWithCourseData from './TableWithCourseData'
 
-const SectionPerYear = ({ thisYearAnalyses, koppsData, year, tableLabels, userLang }) => {
+const AnalysesFromCanvas = ({ thisYearAnalyses, koppsData, tableLabels, userLang }) => {
   const { koppsDataLang } = koppsData
```

Comment:
There are some unused props and variables here. 
```suggestion
const AnalysesFromCanvas = ({ thisYearAnalyses }) => {
```

Reply:
But they might but used in the final implementation later, in that case it is ok to leave the, now.

Reply:
They will be needed to get the URL for Kursplan och Kurs-PM, so I'll leave them for now 👍